Funcky.Async/Extensions/AsyncEnumerableExtensions/SlidingWindow.cs

```diff
@@ -0,0 +1,43 @@
+using System.Runtime.CompilerServices;
+using Funcky.Internal;
+
+namespace Funcky.Async.Extensions
+{
+    public static partial class AsyncEnumerableExtensions
+    {
+        /// <summary>
+        /// SlidingWindow returns a sequence of sliding windows of the given width.
+        /// The nth sequence will start with the nth element of the source sequence.
+        /// </summary>
+        /// <remarks>
+        /// The returned windows always have  'width' many elements.
+        /// i.e. if your source Sequence is smaller than the window, there will be an empty result.
+        /// </remarks>
+        /// <param name="source">The source sequence.</param>
+        /// <param name="width">The width of the sliding window.</param>
+        /// <typeparam name="TSource">The type of the source elements.</typeparam>
+        /// <returns>Returns a sequence of equally sized window sequences.</returns>
+        [Pure]
+        public static IAsyncEnumerable<IReadOnlyList<TSource>> SlidingWindow<TSource>(this IAsyncEnumerable<TSource> source, int width)
+            => SlidingWindowEnumerable(source, ValidateWindowWidth(width));
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static async IAsyncEnumerable<IReadOnlyList<TSource>> SlidingWindowEnumerable<TSource>(IAsyncEnumerable<TSource> source, int width)
+        {
+            var slidingWindow = new SlidingWindowQueue<TSource>(width);
+            await foreach (var element in source)
```

Comment:
Missing `ConfigureAwait(false)`

Reply:
The static analyser does not suggest this one for some reason:
Fixed in c906c8d18a218d9547009809401d17e2b06ce753